2025-10-01 02:20:18 INFO root: Logging to /mnt/eapp/code/transcoder/core/api/logs/api-20251001-022018.log
2025-10-01 02:20:18 INFO src: Ensured admin account 'admin' (id=1)
2025-10-01 02:20:18 INFO werkzeug: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.1.150:5001
2025-10-01 02:20:18 INFO werkzeug: [33mPress CTRL+C to quit[0m
2025-10-01 02:20:18 ERROR src.services.transcoder_client: Transcoder service request failed: HTTPConnectionPool(host='localhost', port=5003): Max retries exceeded with url: /transcode/status (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f8d97ddbeb0>: Failed to establish a new connection: [Errno 111] Connection refused'))
2025-10-01 02:20:18 INFO werkzeug: 192.168.1.23 - - [01/Oct/2025 02:20:18] "[35m[1mGET /transcode/status HTTP/1.1[0m" 503 -
